src/styles/utils/_variables.scss

```diff
@@ -1,4 +1,4 @@
-$daysPerRow: 7;
+$daysPerRow: 8;
```

Comment:
what-why?
7 days was perfectly correct
```suggestion
$daysPerRow: 7;
```

Reply:
Ooops forgot to revert. Was testing. Thanx